### PR TITLE
Threddsclient

### PR DESCRIPTION
**Pull request by cehbrecht**
Checklist

- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ x] Source is from official source
- [ x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ x] If static libraries are linked in, the license of the static library is packaged.
- [ x] Build number is 0
- [ x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/threddsclient/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d9d2e5292a73a9be47b3fc06d0fa306f7cc9f9774acc72c2fee1b89d80fb071b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/threddsclient/meta.yaml]: Can this be `noarch: python`?

Comment (cehbrecht) [reply recipes/threddsclient/meta.yaml]: right. I have updated the recipe. Thanks.

**Comment by mariusvniekerk** [comment]
@davidcaron Are you okay with being a maintainer on this recipe?

**Comment by davidcaron** [comment]
Yes, thanks

**Pull request merged**